recipes/python-pypi-mirror/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "python-pypi-mirror" %}
+{% set version = "5.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python-pypi-mirror-{{ version }}.tar.gz
+  sha256: 65938fad5a9fc71d5f3f436901b3f3be69c176082fd89ee22ed60520476b2659
+
+build:
+  entry_points:
+    - pypi-mirror=pypi_mirror:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - pypi_mirror
+  commands:
+    - pip check
+    - pypi-mirror --help
+  requires:
+    - git
```

Comment:
Why is this only required for the tests?

Reply:
There was a new release since I submitted this removed the git requirement entirely.